### ISSUE TITLE
Unhandled: Unexpected token ...

### ISSUE THREAD

**Issue by najadojo** [Dinomite-Studios/unity-azure-pipelines-tasks#199]
Our pipeline updated to version 3.1.13 and we've been unable to get a successful build. The UnityBuildTask@3 is failing with an Unexpected token error:

```
2022-06-17T20:44:18.9621068Z ##[section]Starting: Build Unity project
2022-06-17T20:44:18.9703341Z ==============================================================================
2022-06-17T20:44:18.9703571Z Task         : Unity Build
2022-06-17T20:44:18.9703785Z Description  : Build a Unity project and get the exported output files.
2022-06-17T20:44:18.9703985Z Version      : 3.1.13
2022-06-17T20:44:18.9704140Z Author       : Dinomite Studios
2022-06-17T20:44:18.9704484Z Help         : Use this task to build your Unity project and generate the target platform's output files. [More Information](https://unitydevops.com/docs/unity-build-task)
2022-06-17T20:44:18.9705025Z ==============================================================================
2022-06-17T20:44:18.9874074Z ##[debug]Using node path: C:\vss-agent\2.204.0\externals\node\bin\node.exe
2022-06-17T20:44:19.1506772Z ##[debug]agent.TempDirectory=D:\a\_work\_temp
2022-06-17T20:44:19.1520606Z ##[debug]loading inputs and endpoints
2022-06-17T20:44:19.1525188Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
2022-06-17T20:44:19.1557191Z ##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
2022-06-17T20:44:19.1559575Z ##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
2022-06-17T20:44:19.1563514Z ##[debug]loading INPUT_BUILDSCRIPTTYPE
2022-06-17T20:44:19.1564688Z ##[debug]loading INPUT_BUILDTARGET
2022-06-17T20:44:19.1565839Z ##[debug]loading INPUT_INLINEBUILDSCRIPT
2022-06-17T20:44:19.1567062Z ##[debug]loading INPUT_OUTPUTFILENAME
2022-06-17T20:44:19.1568086Z ##[debug]loading INPUT_OUTPUTPATH
2022-06-17T20:44:19.1569139Z ##[debug]loading INPUT_UNITYEDITORSPATHMODE
2022-06-17T20:44:19.1570109Z ##[debug]loading INPUT_UNITYPROJECTPATH
2022-06-17T20:44:19.1573240Z ##[debug]loading SECRET_BUILDAGENTACCESSTOEDTPACKAGING
2022-06-17T20:44:19.1575823Z ##[debug]loading SECRET_EDTVSTSACCESS
2022-06-17T20:44:19.1577146Z ##[debug]loading SECRET_SYSTEM_ACCESSTOKEN
2022-06-17T20:44:19.1580958Z ##[debug]loaded 13
2022-06-17T20:44:19.1595000Z ##[debug]Agent.ProxyUrl=undefined
2022-06-17T20:44:19.1596065Z ##[debug]Agent.CAInfo=undefined
2022-06-17T20:44:19.1596431Z ##[debug]Agent.ClientCert=undefined
2022-06-17T20:44:19.1596805Z ##[debug]Agent.SkipCertValidation=undefined
2022-06-17T20:44:19.1636737Z ##[debug]task result: Failed
2022-06-17T20:44:19.1659639Z ##[error]Unhandled: Unexpected token ...
2022-06-17T20:44:19.1664986Z ##[debug]Processed: ##vso[task.issue type=error;]Unhandled: Unexpected token ...
2022-06-17T20:44:19.1666146Z ##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Unexpected token ...
2022-06-17T20:44:19.1684995Z ##[section]Finishing: Build Unity project
```

Let me know if you need more details.

**Comment by waldobronchart** [comment]
I'm also seeing this issue

**Comment by waldobronchart** [comment]
Using v2 seems to alleviate the issue for now

```yml
  - task: UnityBuildTask@2
```

@FejZa Seems like yesterday's update broke things

**Comment by waldobronchart** [comment]
You can specify the previous version like this as a workaround for now

```yml
  - task: UnityBuildTask@3.1.12
```

**Comment by Kostusev** [comment]
Me too. Are there any workaround now for YAML ?

**Comment by gelexgaray** [comment]
The same here. Downgrading to 3.1.12 as says @waldobronchart works as a workaround

**Comment by nemic76** [comment]
I have the same problem. But how can I solve or workaround it with Classic Pipeline Editor?

**Comment by ncupper** [comment]
I have the same problem.

**Comment by gibzi** [comment]
I have the same problem.

**Comment by Reinhartdelille** [comment]
+1

**Comment by binhnguyen86** [comment]
+1

**Comment by KlingOne** [comment]
+1

**Comment by Kostusev** [comment]
@FejZa Can we expect to get  fix for this soon (this week for example)?

**Comment by FejZa** [comment]
@Kostusev yes!

**Comment by Kiranksk** [comment]
+1

**Comment by wiandrade** [comment]
I have the same problem. But how can I solve or workaround it with Classic Pipeline Editor?

**Comment by devichris** [comment]
having the same problem. Hoping a fix can be pushed soon. In the meantime i may pin the version

**Comment by FejZa** [comment]
Fixed in v2.9.0 of the extension package.
